web/client/selectors/context.js

```diff
@@ -95,3 +95,9 @@ export const contextMonitoredStateSelector = createSelector(
  * @param {object} state the app state
  */
 export const resourceSelector = ({context = {}} = {}) => context.resource;
+
+/**
+ * Get the theme configuration for the selected context
+ * @param {object} state the app state
+ */
+export const contextThemeSelector = ({context = {}} = {}) => castArray(context?.currentContext?.theme || []);
```

Comment:
can you add a unit test for this?
maybe castArray is not necessary, but ok

Reply:
about castArray it depends on how the theme will be stored, for now it's only a single object